src/dsp_tools/xmllib/models/resource.py

```diff
@@ -396,3 +398,18 @@ def add_iiif_uri(self, iiif_uri: str, permissions: str | None = None, comment: s
             )
         self.file_value = IIIFUri(iiif_uri, permissions, comment, self.res_id)
         return self
+
+    #######################
+    # Migration Metadata
+    #######################
```

Comment:
why add this here, but not everywhere?  
it's up to you, but I'm not a huge fan of these kinds of comments anymore. In a 400L file, you're unlikely to lose track of where things are.